package.json

```diff
@@ -3,10 +3,13 @@
   "version": "0.1.0",
   "dependencies": {
     "@babel/runtime-corejs2": "^7.3.1",
+    "babel-loader": "^8.0.5",
+    "embarkjs-connector-web3": "4.0.0",
+    "embarkjs-ipfs": "4.0.0",
+    "embarkjs-whisper": "4.0.0",
```

Comment:
These probably need update to `4.0.1+` once its released.

Reply:
Not really a discussion for here, but I feel like we are just adding packages for the sake of it.

I know @iurimatias said that we wouldn't have to add those packages, but looks like we actually do? If it bugs or crashes when those packages are not added, I feel like we should fix it. Otherwise, if it just doesn't enable us to use storage, it's not that big of  a deal, but we'll need to document it correctly that you need to add the required package.